earth_enterprise/SConstruct

```diff
@@ -180,7 +180,24 @@ env.PhonyTargets(**this_dict)
 
 # Clean install dir (note: the current sconscripts leave stuff behind, like
 # directories).
-if env.GetOption('clean'):
-    # clean the search and getomcat install folders.
+
+env.Clean("distclean", [
+  ".sconsign.dblite",
+  ".sconf_temp",
+  "config.log",
+  "src/.sconsign.dblite",
+  "src/.sconf_temp",
+  "src/config.log"
+  ])
+
+if 'buildclean' in COMMAND_LINE_TARGETS:
```

Comment:
Is the -c option necessary if 'buildclean' is present at all? Looks like "scons buildclean" and "scons -c buildclean" are equivalent?

Reply:
Ok it's actually necessary for distclean but not buildclean

Reply:
Do we want to have consistency? I suppose so, right?

Reply:
I think it's just easiest to call it with the -c ...  if i were to stick the buildclean stuff inside a single Clean() target it wouldn't be able to call the next SConstruct down the line (in the src directory), which does require a -c for buildclean

Reply:
So are you saying, don't worry about buildclean being after the -c option? Not really following what you're saying here...

Reply:
I'm saying, it is consistent as long as you use the -c option. Was that your criticism? It wasn't very clear.

Reply:
If its the only way to do it, then so be it. It just seems more logical to me to only allow buildclean with -c and not on its own

Reply:
The -c in scons denotes something that literally has a Clean target defined in the scons sense of the word, which seems to require that it fit inside a list or list comprehension that produces the files which are to be deleted as part of that Clean. If we want to do things other than delete files in a list, it has to be defined in a different way, which means it is not a scons Clean target, which means it doesn't get a -c.

Reply:
Sounds ok. Approve.